packages/sdks/src/context/types.ts

```diff
@@ -6,7 +6,14 @@ import type { Dictionary, Nullable } from '../types/typescript.js';
 import type { ExtraContextTypes } from './extra-context-types.js';
 
 export type RegisteredComponent = ComponentInfo & {
-  component: any;
+  component: /**
+   * Dynamically load components (currently only available for Svelte SDK)
+   */
+  | {
+        load: () => Promise<any>;
+        fallback?: any;
```

Comment:
We can improve this:

- create a `ComponentReference` type, move it to its own file (`src/context/component-reference-types.ts`)
- add a svelte override for it


this will be great in the future to use each framework's component types: `React.Component` etc., instead of the current `any`

Reply:
awesome idea! i have made the updates